packages/react-core-auth/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "main": "dist/index.js",
   "module": "dist/esm/index.mjs",
-  "react-native": "dist/index.js",
+  "react-native": "src/index.ts",
```

Comment:
Trying to understand this change better

Reply:
Metro bundler will use _src/index.ts_ as the entry point for React Native projects, allows Metro to transpile the TS files directly